### ISSUE TITLE
Several deprecated twisted.protocols.* modules point to old twistedmatrix.com/projects/ webpages

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2213]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)| @dreid reported|
|-|-|
|Trac ID|trac#2213|
|Type|task|
|Created|2006-11-02 18:48:37Z|
|Branch|https://github.com/twisted/twisted/tree/module-web-ref-2213|

```
dns.py:                         'http://twistedmatrix.com/projects/names',
ethernet.py:                         'http://twistedmatrix.com/projects/pair',
http.py:                         'http://twistedmatrix.com/projects/web',
imap4.py:                         'http://twistedmatrix.com/projects/mail',
ip.py:                         'http://twistedmatrix.com/projects/pair',
irc.py:                         'http://twistedmatrix.com/projects/words',
jabber.py:                         'http://twistedmatrix.com/projects/words',
msn.py:                         'http://twistedmatrix.com/projects/words',
nntp.py:                         'http://twistedmatrix.com/projects/news',
oscar.py:                         'http://twistedmatrix.com/projects/words',
pop3.py:                         'http://twistedmatrix.com/projects/mail',
raw.py:                         'http://twistedmatrix.com/projects/pair',
rawudp.py:                         'http://twistedmatrix.com/projects/pair',
smtp.py:                         'http://twistedmatrix.com/projects/mail',
sux.py:                         'http://twistedmatrix.com/projects/web',
toc.py:                         'http://twistedmatrix.com/projects/words',
```

We should fix the links or look at removing these long deprecated stubs.  This causes people to follow the API docs outside of the trac webspace and consequently provide no real information.  These documentation also don't point to the new namespaces of the module so they can be found in the API docs.  

<details><summary>Searchable metadata</summary>

```
trac-id__2213 2213
type__task task
reporter__dreid dreid
priority__normal normal
milestone__ 
branch__branches_module_web_ref_2213 branches/module-web-ref-2213
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1162493317000000 1162493317000000
changetime__1220608924000000 1220608924000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph set owner to @dreid|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

What is exactly the problem here? Is this ticket still relevant?

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

an url like http://twistedmatrix.com/projects/words should really be http://twistedmatrix.com/trac/wiki/TwistedWords instead

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

(In [24676]) Branching to 'module-web-ref-2213'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Looks good, go ahead and merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_8"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [24680]) Merge module-web-ref-2213: Replace old links for deprecated modules in twisted.protocols

Author: thijs
Reviewer: exarkun
Fixes: [#2213](https://github.com/twisted/twisted/issues/2213)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
